plugins/modules/cargo.py

```diff
@@ -58,15 +58,87 @@
     type: bool
     default: false
     version_added: 7.5.0
+  force:
+    description:
+      - Forcefully install package(s).
+      - Ignores many checks, and updates/installs forcefully.
+    type: bool
+    default: false
+    required: false
+    version_added: 8.4.0
   state:
     description:
       - The state of the Rust package.
     required: false
     type: str
     default: present
     choices: [ "present", "absent", "latest" ]
+  features:
+    description:
+      - List of features to enable for the package.
+      - Do not use with multiple names.
+    type: list
+    elements: str
+    required: false
+    default: []
+    version_added: 8.4.0
+  all_features:
+    description:
+      - Enable all features for package(s).
+    type: bool
+    default: false
+    required: false
+    version_added: 8.4.0
+  directory:
+    description:
+      - Path to install the package(s) from.
+    type: path
+    required: false
+    version_added: 8.4.0
+  registry:
+    description:
+      - Registry to install the package(s) from.
+    type: str
+    required: false
+    version_added: 8.4.0
+  git:
+    description:
+      - Git URL to install the package(s) from.
+      - Can optionally either be used with C(branch), C(tag) or C(rev).
+    type: str
+    required: false
+    version_added: 8.4.0
+  branch:
+    description:
+      - Git branch to install the package(s) from.
+      - C(git) option must be specified.
+    type: str
+    required: false
+    version_added: 8.4.0
+  tag:
+    description:
+      - Git tag to install the package(s) from.
+      - C(git) option must be specified.
+    type: str
+    required: false
+    version_added: 8.4.0
+  rev:
+    description:
+      - Git commit to install the package(s) from.
+      - C(git) option must be specified.
+    type: str
+    required: false
+    version_added: 8.4.0
+  debug:
+    description:
+      - Specify whether to install package(s) in debug mode.
+    type: bool
+    default: false
+    required: false
+    version_added: 8.4.0
 requirements:
-    - cargo installed
+    - cargo installed, optionally have a toml library like toml or tomli
+      present, if Python version is less than 3.11
```

Comment:
It's not clear to me what you mean with "optionally" when you say "Python version is less than 3.11". Is it required when Python < 3.11? Or what happens if you don't have a TOML library and Python < 3.11?

Reply:
No, I should have been clear. The whole thing is optional, but if you have Python < 3.11, and need TOML support, you need to install `toml` or `tomli`.